### PR TITLE
RDSC-1403-update-docs-for-write-behind-job-changes

### PR DESCRIPTION
**Pull request by yaronp68**

### CODE REVIEW

File: content/rdi/quickstart/write-behind-guide.md [hunk 1]
```diff
@@ -39,16 +39,21 @@ RDI write-behind currently supports these target data stores:
...
 | SQL Server |
 
-
 ## Prerequisites
 
 The only prerequisite for running RDI write-behind is [Redis Gears Python](https://redis.com/modules/redis-gears/) >= 1.2.6 installed on the Redis Enterprise Cluster and enabled for the database you want to mirror to the downstream data store.
 For more information, see [Redis Gears installation]({{<relref "/rdi/installation/install-redis-gears">}}).
 
-## Preparing the write-behind pipeline
+## Preparing the write-behind & read-through pipelines
```
Comment (spicy-sauce) [inline content/rdi/quickstart/write-behind-guide.md:47]: should be only write-behind at this point, read-through should be introduced in the (very near) future.

File: content/rdi/quickstart/write-behind-guide.md [hunk 2]
```diff
@@ -39,16 +39,21 @@ RDI write-behind currently supports these target data stores:
...
 
 - Install [RDI CLI]({{<relref "/rdi/installation/install-rdi-cli">}}) on a Linux host that has connectivity to your Redis Enterprise Cluster.
 - Run the [`configure`]({{<relref "/rdi/reference/cli/redis-di-configure">}}) command to install the RDI Engine on your Redis database, if you have not used this Redis database with RDI write-behind before.
+- **For read-through only** the Redis database proxy has to be reconfigured using the [rladmin CLI](https://docs.redis.com/latest/rs/references/cli-utilities/rladmin/):
+
+  ```bash
+  rladmin> tune db <RDI_BDB_NAME> schedpolicy mnp
+  ```
+
```
Comment (spicy-sauce) [inline content/rdi/quickstart/write-behind-guide.md:56]: should be removed for now.

File: content/rdi/quickstart/write-behind-guide.md [hunk 3]
```diff
@@ -113,9 +118,9 @@ Job definition has the following structure:
 
 ```yaml
 source:
-  keyspace:
-    pattern: emp:*
-    exclude-commands: ["json.del"]
+  redis:
```
Comment (spicy-sauce) [inline content/rdi/quickstart/write-behind-guide.md:121]: `trigger: write-behind` is missing.

**Review by spicy-sauce — CHANGES_REQUESTED** [review]

File: content/rdi/quickstart/write-behind-guide.md (hunk 1 above)
Comment (yaronp68) [reply content/rdi/quickstart/write-behind-guide.md:47]: done

File: content/rdi/quickstart/write-behind-guide.md (hunk 2 above)
Comment (yaronp68) [reply content/rdi/quickstart/write-behind-guide.md:56]: done

File: content/rdi/quickstart/write-behind-guide.md (hunk 3 above)
Comment (yaronp68) [reply content/rdi/quickstart/write-behind-guide.md:121]: done

**Review by spicy-sauce — APPROVED** [review]

File: content/rdi/quickstart/write-behind-guide.md [hunk 4]
```diff
@@ -139,21 +140,25 @@ output:
 
 ### Source section
 
-The `source` section describes the source of data change events in Redis:
+The `source` section describes the source of data in the pipeline.
 
-- The `pattern` attribute specifies the pattern of Redis keys to listen on. The pattern has to correspond to keys that are of Hash or JSON value.
+The `redis` section is common for every pipeline initiated by event in Redis such as applying changes to data. In the case of write-behind it has the information required to activate a pipeline dealing with changes to data. it includes the following attributes:
```
Comment (elena-kolevska) [inline content/rdi/quickstart/write-behind-guide.md:145]:
```suggestion
The `redis` section is common for every pipeline initiated by an event in Redis, such as applying changes to data. In the case of write-behind, it has the information required to activate a pipeline dealing with changes to data. It includes the following attributes:
```

File: content/rdi/quickstart/write-behind-guide.md [hunk 5]
```diff
@@ -139,21 +140,25 @@ output:
...
 
-- The `exclude-commands` attribute specifies which commands not to act on. For example, if you listen on key pattern that has Hash values, you can exclude the `HDEL` command so no deletions of data will propagate to the downstream database. If you don't specify this attribute, RDI write-behind acts on all relevant commands.
+- The `key_pattern` attribute specifies the pattern of Redis keys to listen on. The pattern has to correspond to keys that are of Hash or JSON value.
 
-- The `row_format` attribute can be used with the `full` value in order to receive both the `before` and `after` sections of the payload. Note that for Write Behind events the `before` value of the key is never provided.
+- The `exclude_commands` attribute specifies which commands not to act on. For example, if you listen on key pattern that has Hash values, you can exclude the `HDEL` command so no deletions of data will propagate to the downstream database. If you don't specify this attribute, RDI write-behind acts on all relevant commands.
```
Comment (elena-kolevska) [inline content/rdi/quickstart/write-behind-guide.md:149]:
```suggestion
- The `exclude_commands` attribute specifies which commands not to act on. For example, if you listen on a key pattern with Hash values, you can exclude the `HDEL` command so no data deletions will propagate to the downstream database. If you don't specify this attribute, RDI write-behind acts on all relevant commands.
```

**Pull request merged**